_episodes_rmd/06-vector-open-shapefile-in-r.Rmd

```diff
@@ -178,9 +185,8 @@ ggplot() +
 > > ```
 > > We also check the CRS and extent of each object: 
 > > ```{r}
-> > st_crs(lines_HARV)
+> > st_crs(32618)$proj4string
```

Comment:
Same as comment above, what happens if you do `st_crs(lines_HARV)$proj4string` instead?